#

float32 r
float32 rdot
float32 theta
float32 thetadot
float32[] covariance

